docs/sites.yml

```diff
@@ -3436,3 +3436,53 @@
   built_by: Michael Uloth
   built_by_url: https://www.michaeluloth.com
   featured: false
+- title: NARCAN
+  main_url: https://www.narcan.com
+  url: https://www.narcan.com
+  description: >
+    {titleofthesite} Nasal Spray is the first and only FDA-approved nasal form of naloxone for the emergency treatment of a known or suspected opioid overdose.
```

Comment:
`{titleofthesite}`

You forgot to remove/change these placeholders in your entries.

Reply:
ahh, nice catch. I was under the impression it would reference `title` when using `{titleofthesite}`